### PR TITLE
Add io.github.pwr_solaar.solaar

### PR DESCRIPTION
**Pull request by hadess**
Deps file generated with:
```
flatpak-pip-generator evdev\>=1.1.2 pyudev\>=0.13 PyYAML\>=3.12 python-xlib\>=0.27 psutil\>=5.4.3
flatpak-json2yaml.py python3-modules.json
```

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. https://github.com/pwr-Solaar/Solaar/issues/1540
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.github.pwr_solaar.solaar.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+modules:
+  - python3-modules.yaml
+  - name: solaar
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app --root=/
+    sources:
+      - type: git
+        url: https://github.com/pwr-Solaar/Solaar.git
```
Comment (hfiguiere) [inline io.github.pwr_solaar.solaar.yaml:21]: you need an explicit revision here

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build io.github.pwr_solaar.solaar

**Comment by barthalion** [comment]
Didn't it use to require udev rules on the host?

**Comment by hadess** [comment]
> Didn't it use to require udev rules on the host?

https://github.com/systemd/systemd/pull/22971

**Comment by barthalion** [comment]
bot, build io.github.pwr_solaar.solaar

**Comment by barthalion** [comment]
```
builddir/files/share/appdata/io.github.pwr_solaar.solaar.appdata.xml: FAILED:
• tag-missing           : <release> required
```

**Comment by hadess** [comment]
> builddir/files/share/appdata/io.github.pwr_solaar.solaar.appdata.xml: FAILED:
> • tag-missing           : <release> required

That was fixed in https://github.com/pwr-Solaar/Solaar/pull/1546 and picked up by a bump

**Pull request closed without merging**